server/config/validate-config-keys.js

```diff
@@ -46,7 +46,9 @@ const sharedConfig = parseConfig( '_shared.json' );
  * invalid. Such a missing value could and likely would
  * cause runtime errors in Calypso
  */
-environmentKeys.forEach( ( [ filename, keys ] ) => {
```

Comment:
That's the exact problem. To be honest, this doesn't break for me because I use the console, I was just trying to be nice to GUI people that reported it - such as @Copons and @aduth 

My understanding is that the other alternatives we have are letting know to people that they should:

- launch the GUI from the Calypso path in the console.
- use the Calypso default node as their OS default node.
- configure their GUIs to use the Calypso default node version.

These are all valid alternatives, with other fragilities: dependency on human recall, or the ability to configure/change that in your OS without breaking other things, etc. I still prefer the approach proposed in this PR, because it's so easy to do and will make life better for everyone. I believe we should adapt/modify/test run/build scripts to work well with GUIs wherever we can to provide a friendlier dev experience.